src/geocat/f2py/fortran/rcm2points.pyf

```diff
@@ -5,21 +5,21 @@ python module rcm2points ! in
     interface  ! in :rcm2points
 	! signature : fo = drcm2points(yi, xi, fi, yo, xo, [xmsg, opt])
         subroutine drcm2points(ngrd,nyi,nxi,yi,xi,fi,nxyo,yo,xo,fo,xmsg,opt,ncrit,kval,ier) ! in :rcm2points:rcm2points.f
-            integer,            depend(fi),					intent(hide)	:: ngrd=1 ! shape(fi,2)
-            integer,            depend(yi),					intent(hide)	:: nyi=shape(yi,1)
-            integer,            depend(yi),					intent(hide)	:: nxi=shape(yi,0)
-            double precision,	dimension(nxi,nyi),				intent(in)	:: yi
-            double precision,	dimension(nxi,nyi),depend(nxi,nyi),		intent(in)	:: xi
-            double precision,	dimension(nxi,nyi,ngrd),depend(nxi,nyi,ngrd),	intent(in)	:: fi
-            integer,            depend(yo), 					intent(hide)	:: nxyo=len(yo)
-            double precision,	dimension(nxyo),	 			intent(in)	:: yo
-            double precision,	dimension(nxyo),depend(nxyo),	 		intent(in)	:: xo
-            double precision,	dimension(nxyo,ngrd),depend(nxyo,ngrd),		intent(out) 	:: fo(nxyo,ngrd)
-            double precision,	optional,					intent(in)	:: xmsg=-99
-            integer,		optional,					intent(in)	:: opt=0
-            integer,		optional,					intent(hide)	:: ncrit
-            integer,		optional,					intent(hide)	:: kval
-            integer,		optional,					intent(hide)	:: ier=0
+            integer,            depend(fi),					                    intent(hide)	:: ngrd=shape(fi,2)
+            integer,            depend(yi),					                    intent(hide)	:: nyi=shape(yi,1)
+            integer,            depend(yi),					                    intent(hide)	:: nxi=shape(yi,0)
+            double precision,	dimension(nxi,nyi),			                    intent(in)	    :: yi
+            double precision,	dimension(nxi,nyi),depend(nxi,nyi),		        intent(in)	    :: xi
+            double precision,	dimension(nxi,nyi,ngrd),depend(nxi,nyi),	    intent(in)	    :: fi
+            integer,            depend(yo), 					                intent(hide)	:: nxyo=len(yo)
+            double precision,	dimension(nxyo),	 			                intent(in)	    :: yo
+            double precision,	dimension(nxyo),depend(nxyo),	 		        intent(in)	    :: xo
+            double precision,	dimension(nxyo,ngrd),depend(nxyo,ngrd),		    intent(out) 	:: fo(nxyo,ngrd)
+            double precision,	optional,					                    intent(in)	    :: xmsg=-99
+            integer,		optional,					                        intent(in)	    :: opt=0
+            integer,		optional,					                        intent(hide)	:: ncrit
+            integer,		optional,					                        intent(hide)	:: kval
+            integer,		optional,					                        intent(hide)	:: ier=0
```

Comment:
the formatting of the right side of this looks very uneven on my display, can you confirm if the whitespace you added was tabs or spaces? Should be spaces.

Reply:
Fixed (Had put tabs)